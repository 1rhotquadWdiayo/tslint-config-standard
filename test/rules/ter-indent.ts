/**
 * This is a comment.
 *
 * ```
 * function test () {
 *   return true
 * }
 * ```
 */
function test () {
   return true
}
