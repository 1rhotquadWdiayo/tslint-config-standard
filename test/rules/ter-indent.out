
ter-indent.ts[11, 1]: Expected indentation of 2 spaces but found 3.
